src/lib.rs

```diff
@@ -186,7 +187,6 @@ fn find_extracted(runner: &Runner, id: u32) -> &Extracted {
         .find(|(i, _)| runner.egraph.find(*i) == id)
         .map(|(_, ext)| ext)
         .expect("Couldn't find matching extraction!")
-        .clone()
```

Comment:
Why don't we need a clone here?

Reply:
I'm honestly not 100% sure. `cargo test` was failing if `.clone()` was left in (CI didn't fail though). I think `.clone()` in this context returns a double reference `&&Extracted` while the function is of type `&Extracted`.

Reply:
@mwillsey could you help?

Reply:
I'm a little confused about the confusion. If the compiler lets you not clone, you don't need to clone.

Reply:
Hash tag rust. Sorry, I don't have rust intuition, only C++ intuition, and in C++, if someone removes a copy, you interrogate them viciously.